### PR TITLE
Adding WDT to nRF9160.

### PR DESCRIPTION
**Pull request by macthestack**
Small changes to make WDT work on the nRF9160.

Tested on my hardware (Actinius Icarus):
* set_lfosc_ticks
* activate
* try_recover
* pet

### CODE REVIEW

File: nrf-hal-common/src/wdt.rs [hunk 1]
```diff
@@ -202,7 +213,14 @@ where
     /// Is the watchdog active?
     #[inline(always)]
     pub fn is_active(&self) -> bool {
-        self.wdt.runstatus.read().runstatus().bit_is_set()
+        cfg_if! {
+            if #[cfg(feature = "9160")]
+            {
```
Comment (jonas-schievink) [inline nrf-hal-common/src/wdt.rs:218]:
```suggestion
            if #[cfg(feature = "9160")] {
```

File: nrf-hal-common/src/wdt.rs [hunk 2]
```diff
@@ -202,7 +213,14 @@ where
...
+                self.wdt.runstatus.read().runstatuswdt().bit_is_set()
```
Comment (jonas-schievink) [inline nrf-hal-common/src/wdt.rs:219]: is the PAC wrong here or is this also called `runstatuswdt` in the datasheet?

File: nrf-hal-common/src/wdt.rs [hunk 3]
```diff
@@ -3,7 +3,18 @@
 //! This HAL implements a basic watchdog timer with 1..=8 handles.
 //! Once the watchdog has been started, it cannot be stopped.
 
-use crate::pac::WDT;
+use cfg_if::cfg_if;
+
+cfg_if! {
+    if #[cfg(feature = "9160")]{
```
Comment (jonas-schievink) [inline nrf-hal-common/src/wdt.rs:9]:
```suggestion
    if #[cfg(feature = "9160")] {
```

File: nrf-hal-common/src/wdt.rs [hunk 4]
```diff
@@ -3,7 +3,18 @@
...
+        use crate::pac::WDT_NS as WDT;
+    }
+    else {
```
Comment (jonas-schievink) [inline nrf-hal-common/src/wdt.rs:12]:
```suggestion
    } else {
```

File: nrf-hal-common/src/wdt.rs (hunk 2 above)
Comment (macthestack) [reply nrf-hal-common/src/wdt.rs:217]: The PAC is right I believe. It appears like this in the datasheet: https://infocenter.nordicsemi.com/pdf/nRF9160_PS_v2.0.pdf - p.357

**Comment by macthestack** [comment]
I'm not sure I responded / approved your suggestions in the right fashion.

Thanks for the review and sorry for not running fmt.

**Comment by jonas-schievink** [comment]
Thanks, looks good!

**Pull request merged**